src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -538,22 +544,54 @@ internal static string AppendChangesToDescription(string prDescription, Build ol
                 else
                 {
                     match = IsSquashedPRCommit.Match(commit.Message);
-                    if (!match.Success)
+                    if (match.Success)
                     {
-                        continue;
+                        prNumber = match.Groups[1].Value;
+
+                        // Squash PR Messages are in the form "Nullable annotate TypeCompilationState and MessageID (#39449)"
+                        // Take the 1st line since it should be descriptive.
+                        comment = commit.Message.Split('\n')[0];
```

Comment:
Can we remove the comment assignment here and just add the `commit.Message.Split('\n')[0]` to Line 576?

Reply:
Can't because `comment` gets set differently whether it is a Squash or Merge commit